src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -299,8 +299,7 @@
                                          @preflight-check-passed?
                                          (boolean (not-empty @qr-view-finder)))
             camera-ready-to-scan?   (and (or (not animated?) @render-camera?)
-                                         show-camera?
-                                         (not @qr-code-succeed?))
```

Comment:
@flexsurfer to resolve one issue i have to remove this condition. could you remember any reason to keep this condition here?